tests/process_graphs/test_array_apply_add.json

```diff
@@ -0,0 +1,49 @@
+{
+  "loadcollection1": {
+    "process_id": "load_collection",
+    "arguments": {
+      "spatial_extent": null,
+      "id": "S2L2A",
+      "temporal_extent": null,
+      "bands": ["B01"]
+    }
+  },
+  "reduce": {
+    "process_id": "reduce_dimension",
+    "arguments": {
+      "data": {
+        "from_node": "loadcollection1"
+      },
+      "dimension": "t",
+      "reducer": {
+        "process_graph": {
+          "array_apply": {
+            "process_id": "array_apply",
+            "arguments": {
+              "data": {
+                "from_parameter": "data"
+              },
+              "process": {
+                "process_graph": {
+                  "add1": {
+                    "process_id": "add",
+                    "arguments": {
+                      "x": {
+                        "from_parameter": "x"
+                      },
+                      "index": {
```

Comment:
`add` does not accept an argument `index`, so this can be removed.

Reply:
done